apps/dolly-backend/src/main/java/no/nav/dolly/service/excel/PersonExcelService.java

```diff
@@ -345,23 +346,25 @@ public Mono<Void> preparePersonSheet(XSSFWorkbook workbook,
         return Mono.empty();
     }
 
-    private List<Object[]> getPersondataRowContents(List<String> hovedpersoner) {
+    private List<Object[]> getPersondataRowContents(List<Testident> hovedpersoner) {
 
         var start = System.currentTimeMillis();
-        var personer = new ArrayList<>(getPersoner(hovedpersoner));
+        var personer = new ArrayList<>(getPersoner(hovedpersoner.stream().map(Testident::getIdent).toList(), hovedpersoner));
```

Comment:
Gjerne ta med linjeskift på .map og .toList, vet vi har det mye ellers i backend for å øke lesbarhet selv om det blir litt flere linjer ut av det 🤓 
`hovedpersoner.stream()`
`.map(Testident::getIdent)`
`.toList()`